app/pages/onboarding/02-create-wallet/create-wallet.tsx

```diff
@@ -12,6 +14,14 @@ import {
 } from '../../../components/onboarding';
 
 export const CreateWallet: React.FC = () => {
+  const dispatch = useDispatch();
+  const history = useHistory();
+
+  const createSoftwareWallet = () => {
+    dispatch(onboardingMnemonicGenerationStep(1_500));
```

Comment:
The [spinner of this step](https://www.figma.com/file/VcXbxT1EwXvE1nUlOiBEES/Wallet?node-id=253%3A988), otherwise in many cases it'd simply flash the screen as it generates the mnemonic in a few ms.